docs/Local Development.md

```diff
@@ -27,3 +29,32 @@ yarn dev  // Runs dev server
 ```
 
 Now you can navigate your browser to: http://0.0.0.0:3000/
+
+## Testing
+React application tests are located in `client/tests`. Express server tests are located in `server/tests`.
+
+### Running tests locally
+
+#### Manually
+The following command will run all the tests once.
+```
+yarn test
```

Comment:
we need to run `yarn install` first. In fact, I think we should add `yarn install` before the `yarn dev` step as well. installing locally should be the first step for local dev.

Reply:
I updated the README with those instructions @spectranaut